manifests/logind.pp

```diff
@@ -0,0 +1,31 @@
+# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
```

Comment:
Perhaps this should use `@api private` so puppet-strings can understand it?

Reply:
Sure thing. I just copied from existing classes in this module... fixed in https://github.com/camptocamp/puppet-systemd/pull/120/commits/ee36d4eb19de6376509e18836c478a41b0d69164.